app/src/main/res/values/strings.xml

```diff
@@ -2,4 +2,12 @@
 <resources>
     <string name="app_name">Cryto Compose</string>
     <string name="error_generic">Unexpected error</string>
+
+    <string name="home_title">Trade Now and Get\nYour Life</string>
+
+    <string name="portfolio_card_total_coin_label">Total Coins</string>
+    <string name="portfolio_card_total_coin">$%s</string>
+    <string name="portfolio_card_today_profit_label">Today\'s Profit</string>
+    <string name="portfolio_card_today_profit">$%s</string>
```

Comment:
I think we can rename this string to be a common format string and reuse it. Or simply just ignore and remove their usages in next PR

Reply:
Yep, @luongvo I already updated on this [PR](https://github.com/nimblehq/jetpack-compose-crypto/pull/12/files) 🚀